Software/Source/Bursttest.py

```diff
@@ -0,0 +1,283 @@
+			PacketType=data["PacketType"]
+			Dbg_Pkt,Acc_Pkt,Temp_Pkt,Batt_Pkt,Brst_Pkt=PacketCheck(PacketType)
+		else:
+			break
+	return Dbg_Pkt,Acc_Pkt,Temp_Pkt,Batt_Pkt,Brst_Pkt
+
+#Function to find the burst interval time
+def bursttime(testtime,burstinterval):
+	burstinterval1=testtime/burstinterval
+
+	return(float(burstinterval1))
+
+# Function to calculate packet type loss
+def losscal(burstinterval1,a,b,c,d,e): 
+    
+	global Dbg_Pkt_loss
+	Dbg_Pkt_loss = (( burstinterval1 - a ) / burstinterval1 ) * 100
+
+	global Acc_Pkt_loss
+	Acc_Pkt_loss = ((( burstinterval1 * 14 ) - b ) /( burstinterval1 * 14 )) * 100
+
+	global Temp_Pkt_loss
+	Temp_Pkt_loss = (( burstinterval1 - c ) / burstinterval1 ) * 100
+
+	global Batt_Pkt_loss
+	Batt_Pkt_loss = (( burstinterval1 - d ) / burstinterval1 ) * 100
+
+	global Brst_Pkt_loss
+	Brst_Pkt_loss = (( burstinterval1 - e ) / burstinterval1 ) * 100
+
+	return(round(Dbg_Pkt_loss,2)),round(Acc_Pkt_loss,2),round(Temp_Pkt_loss,2),round(Batt_Pkt_loss,2),round(Brst_Pkt_loss,2)
+
+#Function to count the packet types in the file and return array of count 
+def indpacketcheck(packettype):
+
+    if packettype == '0':
+        global k
+        k = k + 1
+    elif packettype =='1':
+        global l
+        l = l + 1
+    elif packettype =='2':
+        global m
+        m = m + 1
+
+    elif packettype =='3':
+        global n
+        n = n + 1
+
+    elif packettype =='4':
+        global o
+        o = o + 1
+    return(k,l,m,n,o)
+
+# Function to count the packet types in the file and return array of count 
+def indbrstpktcnt(macid):
+	global k
+	global l
+	global m
+	global n
+	global o
+	brstpktcnt = [ ]
+	tempfilename = str(macid)+".txt"
+	f2 = open(tempfilename,"r")
+	while True:
+		data = f2.readline()
+		if data:
+			data_obj = json.loads(data)
+			packettype = data_obj["PacketType"]
+			k,l,m,n,o = indpacketcheck(packettype)
+			if packettype == '4':
+				# Create array of data
+				brstpktcnt.append(k)
+				brstpktcnt.append(l)
+				brstpktcnt.append(m)
+				brstpktcnt.append(n)
+				brstpktcnt.append(o)
+                # Clear count
+				k = 0
+				l = 0
+				m = 0
+				n = 0
+				o = 0
+		else:
+			break
+	#print(brstpktcnt)
+	return brstpktcnt
+
+
+
+#Create csv file
+def csvfile(file_mac,Test_time,count1,count):
+
+	file_name = str(file_mac)+".csv"
+	with open(file_name,"w") as f:
+		write=csv.writer(f,delimiter=',', lineterminator='\n')
+
+		Test_Time=["Testtime",Test_time]
+		Packet_loss=["Pkt loss %"]
+		Packet_count=["Pkt Cnt"]
+		Packet=[' ','Dbg Pkt','Acc Pkt','Temp Pkt','Batt Pkt','Brst Pkt']
+
+		write.writerow(Test_Time)
+		write.writerow(Packet)
+		write=csv.writer(f,lineterminator= ',')
+		write.writerow(Packet_loss)
+		write=csv.writer(f,lineterminator= '\n')
+		write.writerow(count1)
+		write=csv.writer(f,lineterminator= ',')
+		write.writerow(Packet_count)
+		write=csv.writer(f,lineterminator= '\n')
+		write.writerow(count)
+		write=csv.writer(f,lineterminator= '\n')

```

Comment:
multiple csv.writer function need not be called. Work on the logic to have single csv.writer function